pwa-devdocs/_drafts/project-setup/create-theme-files/index.md

```diff
@@ -53,7 +53,7 @@ In this part of the [Project setup tutorial], you will create the initial files
         "require": {
             "php": "~7.1.0|~7.2.0",
             "magento/framework": "100.3.*",
-            "magento-research/theme-frontend-pwa": "*"
+            "magento/theme-frontend-pwa": "*"
```

Comment:
❌ This has not moved and the original repository is read only. Should we drop this?